content/blog/2019-01-06-babette.md

```diff
@@ -0,0 +1,205 @@
+---
+slug: babette
+title: Call BEAST2 from R
+package_version: 2.0
+authors:
+  - Richèl J.C. Bilderbeek
+date: 2019-01-06
+categories: blog
+topicid: 
+tags:
+  - Software Peer Review
+  - R
+  - community
+  - software
+  - packages
+  - phylogeny
+  - phylogenetics
+  - BEAST
+  - BEAST2
+  - Bayesian
+---
+
+`babette` [^1] is a package to work with BEAST2 [^2], 
+a software platform for Bayesian evolutionary analysis from R.
+
+`babette` is a spin-off of my own academic research.
+As a PhD I work on models of diversification: mathematical descriptions
+of how species form new species. Instead of working on a species'
+individuals, I work on species as evolutionary lineages.
+A good way to show the evolutionary relationships between species
+are phylogenies. For example, this a phylogeny, with four species,
+eloquently named '1', '2', '3' and '4':
+
+![A phylogeny with four species](img/blog-images/2019-01-06-babette/phylogeny.png)
+
+In my research, I simulate thousands of these. I call these 
+phylogenies 'the truth'. Then I wonder, would such a phylogeny actually
+be true, how well would we be able to infer it? I say 'infer', instead
+of 'measure', as we cannot measure phylogenies directly in nature. 
+
+In nature, however, we can sequence the DNA of species. From these DNA
+sequences, we use software to neatly arrange these sequences, creating
+a DNA alignment. Such DNA alignments are where most researchers start from.
+Here is such an alignment, of four (eloquently named) species
+and their aligned DNA sequences, in which a different color represents a
+different nucleotide:
```

Comment:
```suggestion
different nucleotide (an individual element of a DNA sequence):
```

I suggest a (generic?) definition of "nucleotide" because many readers won't know, and the rest of your explanation is so beautifully clear.